tests/test_cloudwatch.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from dataclasses import dataclass
+from datetime import date, datetime
+from fondat.aws import Client, Config
+from fondat.aws.cloudwatch import cloudwatch_resource
+from fondat.error import NotFoundError
+from fondat.monitoring import Measurement, Counter, Gauge, Absolute
+
+pytestmark = pytest.mark.asyncio
+
+
+config = Config(
+    endpoint_url="http://localhost:4566",
+    aws_access_key_id="id",
+    aws_secret_access_key="secret",
+    region_name="us-east-1",
+)
+
+
+@pytest.fixture(scope="function")
+async def client():
+    async with Client(service_name="cloudwatch", config=config) as client:
+        yield client
+
+
+@pytest.fixture(scope="function")
+async def metirc_type(client):
+    _now = lambda: datetime.now()
```

Comment:
FYI, `datetime.now()` will get local time not UTC. To get UTC, should be `datetime.now(tz=timezone.utc)`.